xml_inscriptions/transcribed/MI.AA.UM.KM.L.998.xml

```diff
@@ -46,8 +48,8 @@
                         </layout>
                      </layoutDesc>
                   </objectDesc>
-                  <handDesc>
-                     <handNote/>
+                  <handDesc hands="2">
```

Comment:
I may have missed it below, but normally when hands change in an inscription you mark that using `<handShift/>`.